src/windows/osd.c

```diff
@@ -402,65 +402,111 @@ ssize_t sendmsg(SOCKET sd, struct msghdr *msg, int flags)
 	goto fn_complete;
 }
 
+void sort_list_of_adapters(PIP_ADAPTER_ADDRESSES head)
+{
+	PIP_ADAPTER_ADDRESSES ptr = head, tmp = NULL, prev = NULL;
+	bool sorted = false;
+	if (!head) {
+		return;
+	}
+	do {
+		sorted = false;
+		ptr = head;
+		while (ptr->Next) {
```

Comment:
The while() would be better as a for() loop:
```
for (ptr = head; ptr->Next; ptr = ptr->Next)
```

Reply:
@shefty Done.